src/sensor_state_data/library.py

```diff
@@ -44,31 +36,21 @@ class SensorLibrary:
         device_class=SensorDeviceClass.GAS,
         native_unit_of_measurement=Units.VOLUME_CUBIC_METERS,
     )
-    BATTERY__PERCENTAGE = BaseSensorDescription(
-        device_class=SensorDeviceClass.BATTERY,
+    HUMIDITY__PERCENTAGE = BaseSensorDescription(
+        device_class=SensorDeviceClass.HUMIDITY,
         native_unit_of_measurement=Units.PERCENTAGE,
     )
-    PM1__CONCENTRATION_MICROGRAMS_PER_CUBIC_METER = BaseSensorDescription(
-        device_class=SensorDeviceClass.PM1,
-        native_unit_of_measurement=Units.CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-    )
-    PM10__CONCENTRATION_MICROGRAMS_PER_CUBIC_METER = BaseSensorDescription(
-        device_class=SensorDeviceClass.PM10,
-        native_unit_of_measurement=Units.CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-    )
-    PM25__CONCENTRATION_MICROGRAMS_PER_CUBIC_METER = BaseSensorDescription(
-        device_class=SensorDeviceClass.PM25,
-        native_unit_of_measurement=Units.CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    KEG_SIZE__VOLUME_LITERS = BaseSensorDescription(
+        device_class=SensorDeviceClass.KEG_SIZE,
+        native_unit_of_measurement=Units.VOLUME_LITERS,
     )
-    CO2__CONCENTRATION_PARTS_PER_MILLION = BaseSensorDescription(
-        device_class=SensorDeviceClass.CO2,
-        native_unit_of_measurement=Units.CONCENTRATION_PARTS_PER_MILLION,
+    KEG_TYPE__NONE = BaseSensorDescription(
```

Comment:
Are these unrelated changes?

Reply:
This is just a reordering of this list in alphabetical order, See the first post